packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -117,13 +112,7 @@ export const buildModules = async (
           fileInfo.name
         ),
       },
-      esbuild: {
-        logOverride: {
-          "css-syntax-error": "silent",
```

Comment:
We can keep this if necessary. We're you still getting the warnings after I couldn't reproduce yesterday?